src/core/Form/Textarea/Textarea.test.tsx

```diff
@@ -276,4 +276,16 @@ describe('props', () => {
       expect(container.firstChild).toHaveClass('fi-textarea--full-width');
     });
   });
+
+  describe('containerProps', () => {
+    it('has the given props on the container', () => {
+      const { container } = render(
+        <Textarea
+          labelText="label"
+          containerProps={{ style: { width: 100 } }}
```

Comment:
Is this without unit intentionally?

Reply:
Yes, it is. It is now done like React assumes the unit to be `px` in this case. 
It can be done also like this:
`containerProps={{ style: { width: '100px' } }}`

Reply:
Which would be more common use case? We have used values with units previously I think.

Reply:
`TextInput` and `SearchInput` seems to be the only places where is similar situation and it does tell the width as string to explicitly tell the unit to be `px`.

I will change these occurrences to use the string format also.